waspc/src/Wasp/Analyzer/Parser/Lexer.x

```diff
@@ -79,17 +79,19 @@ startCodeToInt (QuoterStartCode _) = quoter
 --
 --   This function is taken from the Alex basic wrapper.
 alexGetByte :: AlexInput -> Maybe (Word8, AlexInput)
-alexGetByte (c, (b:bs), s) = Just (b, (c, bs, s))
-alexGetByte (_, [], []) = Nothing
-alexGetByte (_, [], (c:s)) = case encodeChar c of
-                               (b:bs) -> Just (b, (c, bs, s))
-                               [] -> Nothing
+alexGetByte (prevChar, (currChar, (b:bs)), remainingSource) =
```

Comment:
I thought I need this `currChar` here for something. It gets removed in the later PRs.